source/write/update.txt

```diff
@@ -0,0 +1,220 @@
+
+- **Query filter**, which matches which documents to update. To learn
+  more about query filters, see the :ref:`kotlin-sync-specify-query`
+  guide.
+- **Update document**, which specifies the update operator, or the kind of update to
+  perform, and the fields and values that should change. For a list of update
+  operators and their usages, see the :manual:`Field Update Operators
+  guide page</reference/operator/update-field/>` in the {+mdb-server+} manual.
+
+Update One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``updateOne()`` method to update the
+``name`` value of a document from ``"Happy Garden"`` to ``"Mountain
+House"``:
+
+.. literalinclude:: /includes/write/update.kt
+   :start-after: start-update-one
+   :end-before: end-update-one
+   :language: kotlin
+   :copyable:
+
+Update Many Documents
+~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``updateMany()`` method to update all documents
+in which the ``name`` value is ``"Starbucks"``. The update renames the
+``address`` field to ``location``.
+
+.. literalinclude:: /includes/write/update.kt
+   :start-after: start-update-many
+   :end-before: end-update-many
+   :language: kotlin
+   :copyable:
+
+Customize the Update Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``updateOne()`` and ``updateMany()`` methods optionally accept
+a parameter that sets options to configure the update operation.
+If you don't specify any options, the driver performs update
+operations with default settings.
+
+The following table describes the setter methods that you can use to
+configure an ``UpdateOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``upsert()``
+     - | Specifies whether the update operation performs an upsert operation if no 
+         documents match the query filter. For more information, see the :manual:`upsert
+         statement </reference/command/update/#std-label-update-command-upsert>`
+         in the {+mdb-server+} manual.
+       | Defaults to ``false``
+
+   * - ``bypassDocumentValidation()``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``false``.
+
+   * - ``collation()``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``arrayFilters()``
+     - | Contains a list of filters that specifies which array elements an update applies
+         to.
+
+   * - ``hint()``
+     - | Gets or sets the index to scan for documents. 
```

Comment:
Q: The intro to this section refers to this collection of methods as "setter methods" but this method in particular mentions getting OR setting. Does it make sense to discuss the get functionality of this method within the context of this table?

Reply:
Ah, copypasta error. This is only a `set` method in this driver, `getHint()` would be the getter. Will review the descriptions.